rust/datafusion/src/execution/context.rs

```diff
@@ -495,13 +495,30 @@ impl QueryPlanner for DefaultQueryPlanner {
     }
 }
 
+/// The style of case display
+#[derive(Clone, PartialEq)]
+pub enum CaseStyle {
+    /// Case style like MySQL, SELECT mD5("a") will output SELECT mD5("a")
+    LikeMySQL,
+    /// Case style like PostgreSQL, SELECT mD5("a") will output SELECT md5("a")
+    LikePostgreSQL,
```

Comment:
```suggestion
    /// function names in the output are displayed as they were provided:
    /// 'mD5("a")' will appear in the results as `mD5("a")`. This 
    /// mimics MySQL behavior
    PreserveCase,
    /// function names in the output are displayed in lower case:
    /// `mD5("a")` will appear in the results as  `md5("a")`This 
    /// mimics PostgreSQL behavior
    LikePostgreSQL,
```

I think naming the enum variants based on what they do may be clearer